frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -74,63 +81,75 @@ const LogControls: React.FC<LogControlsProps> = ({
+
   return (
-    <div className="co-toolbar">
-      <div className="co-toolbar__group co-toolbar__group--left">
-        <div className="co-toolbar__item">
-          <SelectDeprecated
-            aria-label={t('public~Select a path')}
-            onToggle={onTogglePath}
-            onSelect={onChangePath}
-            selections={path}
-            isOpen={isPathOpen}
-          >
-            {options(pathItems)}
-          </SelectDeprecated>
-        </div>
-        {isJournal && <NodeLogsFilterUnit onChangeUnit={onChangeUnit} unit={unit} />}
-        {!isJournal && (
-          <div className="co-toolbar__item">
-            {isLoadingFilenames ? (
-              <LoadingInline />
-            ) : (
-              logFilenamesExist && (
-                <SelectDeprecated
-                  aria-label={logLabel}
-                  placeholderText={logLabel}
-                  onToggle={onToggleFilename}
-                  onSelect={onChangeFilename}
-                  selections={logFilename}
-                  isOpen={isFilenameOpen}
-                  className="co-node-logs__log-select"
+    <Toolbar className="co-toolbar-empty-state">
+      <ToolbarContent>
+        <ToolbarGroup>
+          <ToolbarItem>
+            <Select
+              onSelect={onChangePath}
+              selected={path}
+              isOpen={isPathOpen}
+              toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                <MenuToggle
+                  ref={toggleRef}
+                  onClick={onTogglePath}
+                  aria-label={t('public~Select a path')}
                 >
-                  {options(logFilenames)}
-                </SelectDeprecated>
-              )
-            )}
-          </div>
-        )}
-        {showSearch && (
-          <div className="co-toolbar__item">
-            <LogViewerSearch placeholder={t('public~Search')} minSearchChars={0} />
-          </div>
-        )}
-      </div>
-      <div className="co-toolbar__group co-toolbar__group--right">
-        <div className="co-toolbar__item">
+                  {path}
+                </MenuToggle>
+              )}
+            >
+              <SelectList>{options(pathItems)}</SelectList>
+            </Select>
+          </ToolbarItem>
+          {isJournal && <NodeLogsFilterUnit onChangeUnit={onChangeUnit} unit={unit} />}
+          {!isJournal && (
+            <ToolbarItem>
+              {isLoadingFilenames ? (
+                <LoadingInline />
+              ) : (
+                logFilenamesExist && (
+                  <Select
+                    onSelect={onChangeFilename}
+                    selected={logFilename}
+                    isOpen={isFilenameOpen}
+                    className="co-node-logs__log-select"
+                    toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                      <MenuToggle ref={toggleRef} onClick={onToggleFilename} aria-label={logLabel}>
```

Comment:
`aria-label` shouldn't be needed since the toggle label will indicate value.
```suggestion
                      <MenuToggle ref={toggleRef} onClick={onToggleFilename}>
```